restructuredtext_lint/cli.py

```diff
@@ -5,20 +5,36 @@
 import os
 import sys
 
+from collections import OrderedDict
+
+from docutils.utils import Reporter
+
 from restructuredtext_lint.lint import lint_file
 
+# Generate our levels mapping for humans, using ordered dict for --help string
+# http://repo.or.cz/docutils.git/blob/422cede485668203abc01c76ca317578ff634b30:/docutils/docutils/utils/__init__.py#l65
+LEVEL_MAP = OrderedDict([
+    ('debug', Reporter.DEBUG_LEVEL),  # 0
+    ('info', Reporter.INFO_LEVEL),  # 1
+    ('warning', Reporter.WARNING_LEVEL),  # 2
+    ('error', Reporter.ERROR_LEVEL),  # 3
+    ('severe', Reporter.SEVERE_LEVEL),  # 4
+])
+
+
 # Load in VERSION from standalone file
 with open(os.path.join(os.path.dirname(__file__), 'VERSION'), 'r') as version_file:
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, level=2):
```

Comment:
Please use `level=LEVEL_MAP.get('warning')` instead of `level=2` so we have 1 less magic number